ios/FluentUI.Demo/FluentUI.Demo/Demos/SearchBarDemoController.swift

```diff
@@ -99,8 +99,17 @@ class SearchBarDemoController: DemoController, SearchBarDelegate {
         let dataSource = BadgeViewDataSource(text: text, customView: customView)
         let badge = BadgeView(dataSource: dataSource)
         badge.lineBreakMode = .byTruncatingTail
-        badge.disabledBackgroundColor = UIColor(colorValue: GlobalTokens.sharedColors(.purple, .primary))
-        badge.disabledLabelTextColor = .white
+        var customTokens: [BadgeViewTokenSet.Tokens: ControlTokenValue] {
+            return [
+                .backgroundDisabledColor: .dynamicColor {
+                    return DynamicColor(light: GlobalTokens.sharedColors(.purple, .primary))
+                },
+                .foregroundDisabledColor: .dynamicColor {
+                    return DynamicColor(light: GlobalTokens.neutralColors(.white))
+                }
+            ]
+        }
+        badge.tokenSet.replaceAllOverrides(with: customTokens)
```

Comment:
You could also do
```
badge.tokenSet[.backgroundDiabledColor] = .dynamicColor { .init(light: GlobalTokens.sharedColors(.purple, .primary) }
badge.tokenSet[.foregroundDisabledColor = .dynamicColor { .init(light: GlobalTokens.neutralColors(.white) }
```
Which I would prefer, because I think we should be showing clients that you can set the overrides individually, rather than just through the bulk `.replaceAllOverrides(with: )` method.